.github/workflows/changelog.yml

```diff
@@ -37,9 +39,8 @@ jobs:
           PR_COMMENT_BODY: ${{ github.event.comment.body }}
       - name: Commit and push
         run: |
-          # use the last commit author name and email
-          git config --global user.name "$(git --no-pager log --format=format:'%an' -n 1)"
-          git config --global user.email "$(git --no-pager log --format=format:'%ae' -n 1)"
+          git config --global user.email "support+ci@expo.io"
```

Comment:
Will this make PRs partially authored by the bot?

Reply:
I think it might be the case, although in another repo I was only testing it using my account PAT, so I'm not 100% sure